domains/peachy.json

```
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "Peachysy",
+           "email": "minecraft350star@gmail.com",
+           "discord": "781577857511587900"
+        },
+    
+        "record": {
+            "CNAME": "https://peachysy.github.io/"
```

```suggestion
            "CNAME": "peachysy.github.io"
```
Also maybe consider making the text black? (Since the color of the text is white and its blending in the background)